### PR TITLE
Add a comment on canonical URL

### LINKED ISSUE #258: Add a comment on top of the list, asking people to pull the list from the right URL
With MXR gone, code like these are now broke.

https://github.com/search?q=https%3A%2F%2Fmxr.mozilla.org%2Fmozilla-central%2Fsource%2Fnetwerk%2Fdns%2Feffective_tld_names.dat&ref=searchresults&type=Code&utf8=%E2%9C%93

We should add a minimum comment on top of the list, ask people writes new code to pull the file from https://publicsuffix.org/list/public_suffix_list.dat

### PR DESCRIPTION
**Pull request by timdream**
Fixes #258

### CODE REVIEW

**Comment by weppos** [comment]
It seems reasonable to me. I'll merge it unless any objection from @gerv.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -2,6 +2,12 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+// Please pull this list from, and only from https://publicsuffix.org/list/public_suffix_list.dat , 
+// rather than any other VCS sites. These sites are not intended to be used that way and will likely
+// be gone in the future.
+
```
Comment (gerv) [inline public_suffix_list.dat]: I'd remove the second sentence and replace it with "Pulling from any other URL is not guaranteed to be supported."

**Comment by gerv** [comment]
With that fix, r=gerv.

**Comment by timdream** [comment]
@gerv Done.

**Comment by timdream** [comment]
@weppos Could you merge this?

**Pull request merged**